functions.php

```diff
@@ -133,3 +133,44 @@ function twentytwentyfive_pattern_categories() {
 endif;
 
 add_action( 'init', 'twentytwentyfive_pattern_categories' );
+
+add_action( 'init', 'twentytwentyfive_register_block_bindings', 9 );
+
+/**
+ * Register block bindings.
+ */
+if ( ! function_exists( 'twentytwentyfive_register_block_bindings' ) ) :
+	/**
+	 * Register the copyright block binding source.
+	 *
+	 * @since Twenty Twenty-Five 1.0
+	 * @return void
+	 */
+	function twentytwentyfive_register_block_bindings() {
+		register_block_bindings_source(
+			'twentytwentyfive/copyright',
+			array(
+				'label'              => __( 'Copyright', 'twentytwentyfive' ),
+				'get_value_callback' => 'twentytwentyfive_copyright_binding',
+			)
+		);
+	}
+	/**
+	 * Callback function for the copyright block binding source.
+	 *
+	 * @since Twenty Twenty-Five 1.0
+	 * @return string Copyright text.
+	 */
+	function twentytwentyfive_copyright_binding() {
+		/* translators: Copyright date format, see https://www.php.net/manual/datetime.format.php */
+		$date_format    = _x( 'Y', 'copyright date format', 'twentytwentyfive' );
```

Comment:
I'm not sure about passing this one through a translation function. 

If it were something other than the year, and it needed a date format it would make sense. I can't think of any language with a different format than the four numbers.